gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/langchain/rag_chain.py

```diff
@@ -166,12 +173,26 @@ def create_rag_chain(query: RagQuery) -> ConversationalRetrievalChain:
         index_name=query.document_index_name,
     )
 
+    # TODO to be removed after tests of flashrank rerank
+    compressor = None
+    if query.document_index_name in application_settings.flashrank_rerank_doc_index_to_compress:
+        flashrank_rerank_params = (
```

Comment:
Add debug log before it indicating that this document_index_name is eligible to Flashrank Rerank.